.eslintrc.js

```diff
@@ -55,6 +55,10 @@ module.exports = {
     'import/unambiguous': 'off', // scripts
     'import/namespace': ['error', { allowComputed: true }],
     'import/no-extraneous-dependencies': 'off',
+    'import/order': ['error', {
```

Comment:
We try to be as unopinionated as possible around the Airbnb rules.
Will that conflict with?
https://github.com/airbnb/javascript/blob/aefff97bd10e8e17963749bd1013a6927d90fe29/packages/eslint-config-airbnb-base/rules/imports.js#L141-L147

Reply:
the double array makes it so [all groups can be mingled](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#groups-array) (other [ref](https://github.com/benmosher/eslint-plugin-import/issues/172))